src/context/reducers/createAnalysisReducer.tsx

```diff
@@ -1,46 +1,27 @@
 import { ActionMap, CreateAnalysisTypes } from "../actions/types";
 
-
-export interface PatientPersonalInfo {
-  patientName: string;
-  patientAge: number;
-  patientBirthdate: string
-  patientGender: string,
-}
-
 export interface SelectionStates {
   patientID?: string;
   selectedStudyUIDs: SelectedStudies;
   currSelectedStudyUID: string;
 }
 
-
 export interface SelectedStudies {
   [uid: string]: {
     [SeriesInstanceUID: string]: string
   }
 }
 
-export type ICreateAnalysisState = PatientPersonalInfo & SelectionStates;
+export type ICreateAnalysisState =  SelectionStates;
```

Comment:
Would it be better to get rid of the `SelectionStates` interface and just define `ICreateAnalysisState` directly as:
```
export interface SelectionStates {
  patientID?: string;
  selectedStudyUIDs: SelectedStudies;
  currSelectedStudyUID: string;
}
```

Reply:
Completely agree, now that we got rid of the other type, it doesn't make sense to have a redundant type like `SelectionStates`